unicorn_mode/samples/persistent/Makefile

```diff
@@ -45,3 +45,6 @@ harness: harness.o
 
 debug: harness-debug.o
 	${MYCC} -L${LIBDIR} harness.o ../../unicornafl/libunicornafl.a $(LDFLAGS) -o harness-debug
+
+fuzz: harness
+	../../../afl-fuzz -m none -i sample_inputs -o out -- ./harness @@
```

Comment:
btw we still didn't add the unicornafl samples as travis test cases, should do that at some point